packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -76,6 +78,9 @@ class MediaQueryData {
   /// the specified font size.
   final double textScaleFactor;
 
+  /// The system insets around the edges of the media (e.g., the screen).
+  final EdgeInsets viewInsets;
```

Comment:
this is the most user facing front document right? Maybe add a bit more details for the nuances between this and padding

Reply:
Good point! Done.